actors/builtin/storage_market/storage_market_actor.go

```diff
@@ -244,79 +232,22 @@ func (a *StorageMarketActor) OnMinerSectorsTerminate(rt Runtime, params *OnMiner
 	return &adt.EmptyValue{}
 }
 
-func (a *StorageMarketActor) OnEpochTickEnd(rt Runtime, _ *adt.EmptyValue) *adt.EmptyValue {
```

Comment:
This is dropped in favor of HandleExpiredDeals, which can be called by anyone to cleanup expiring deals. Doing this allows us to drop a ton of state from the actor, making the market much more efficient.